changelog/61275.removed

```diff
@@ -0,0 +1 @@
+Removed transport kwarg from salt.utils.event.get_event
```

Comment:
Sorry once again for not finding this in the original review. I promise this is the last one:

Since this is a deprecation now and not a removal, please change the file extension to `.deprecated` and update the details of the changelog to include deprecating the transport kwarg not removing

Reply:
All good!  I'll get that changed and pushed now